edx_proctoring/templates/proctored_exam/instructions.html

```diff
@@ -79,12 +79,12 @@ <h2 id="acessible-error-title">{% trans "Cannot Start Proctored Exam" %}</h2>
 </div>
 
 <div class="footer-sequence border-b-0 padding-b-0">
-  {% if not is_sample_attempt %}
-  <p class="proctored-exam-instruction">
+  {% if not is_sample_attempt and allow_unproctored %}
+    <p class="proctored-exam-instruction">
       {% trans "Don't want to take this exam with online proctoring?" %}<a class="proctored-decline-exam" href='#' data-action="decline" data-exam-id="{{exam_id}}" data-change-state-url="{{change_state_url}}">
         {% trans "Take this exam as an open exam instead." %}
```

Comment:
I'm confused about the purpose of this?  Why wouldn't this want similar text?  Is an "open exam" different from an "exam without proctoring"?  Wouldn't this want the same warning about academic credit?  I know this wasn't added by your change, but I think discussing a little clean-up is warranted, even if we decide to do nothing.